docs/integrations/angular.rst

```diff
@@ -110,3 +110,26 @@ Inside your main Angular application module, you need to declare `ngRaven` as a
       'myAppControllers',
       'myAppFilters'
     ]);
+
+Module loaders (CommonJS)
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The raven angular module can be loaded using a module loader like Browserify or Webpack.
+
+.. code-block:: javascript
+    var angular = require('angular');
+    var ngRaven = require('raven-js/plugins/angular').module;
+    var ngRoute = require('angular-route');
+
+    var myAppFilters = require('./myAppFilters');
+    var myAppControllers = require('./myAppControllers');
+    var module = 'myApp'
```

Comment:
Assigning `'myApp'` temporarily to `module` (which is a reserved word kind of) seems silly. If you could remove this line, and just do:

```javascript
angular.module('myApp', [
  ngRaven,
  // ...
]);
```

I'll accept this patch.